gateway/api/models.py

```diff
@@ -35,6 +35,7 @@ class Program(ExportModelOperationsMixin("program"), models.Model):
     arguments = models.TextField(null=False, blank=True, default="{}")
     env_vars = models.TextField(null=False, blank=True, default="{}")
     dependencies = models.TextField(null=False, blank=True, default="[]")
+    config = models.TextField(null=False, blank=True, default="[]")
```

Comment:
Can we make this config option as another model that will be connected to `Program`. Something like
```python
class ProgramConfig(models.Model):
     PY_39: str = "PY_39"
     python_version = models.ChoiceField([PY_39, ...])
     min_workers=model.IntegerField(...)
     ...

class Program(...):
    ...
    config = models.ForeignField(ProgramConfig, on_delete=CASCADE, null=True)
```

This will enable schema verification out of the box. And we will avoid unstructured jsons

Reply:
@IceKhan13 Thanks for review.  I could make a model but this configuration is provider dependent so it may not be good to define a model if it’s not common among providers. Does it make sense?  WDYT?

Reply:
on client side each provider can have different configuration, but for gateway (backend) and gateway provider we need to enforce this, I think.